naturaldateformat/src/main/java/tk/zielony/naturaldateformat/NaturalDateFormat.java

```diff
@@ -1,36 +1,46 @@
 package tk.zielony.naturaldateformat;
 
-import android.content.Context;
-
+import ohos.app.Context;
 import org.joda.time.Chronology;
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
-
 import java.util.TimeZone;
 
 /**
  * Created by Marcin on 29.01.2016.
  */
 public abstract class NaturalDateFormat {
-    public static final int DAYS = 1, MONTHS = 2, HOURS = 4, MINUTES = 8, YEARS = 16, SECONDS = 32, DATE = DAYS | MONTHS | YEARS, TIME = SECONDS | MINUTES | HOURS, WEEKDAY = 64;
+
+    public static final int DAYS = 1;
+    public static final int MONTHS = 2;
+    public static final int HOURS = 4;
+    public static final int MINUTES = 8;
+    public static final int YEARS = 16;
+    public static final int SECONDS = 32;
+    public static final int DATE = DAYS | MONTHS | YEARS;
+    public static final int TIME = SECONDS | MINUTES | HOURS;
+    public static final int WEEKDAY = 64;
 
     protected Context context;
+
     protected int format;
+
     protected TimeZone zone;
+
     protected Chronology chronology;
 
-    public NaturalDateFormat(Context context, int format) {
+    protected NaturalDateFormat(Context context, int format) {
         this.context = context;
         this.format = format;
     }
 
-    public NaturalDateFormat(Context context, Chronology chronology, int format) {
+    protected NaturalDateFormat(Context context, Chronology chronology, int format) {
```

Comment:
keep the constructors as public. Applies to all the parameterized constructors in the class.

Reply:
done